src/components/form-version/row.vue

```diff
@@ -66,6 +65,8 @@ defineEmits(['view-xml']);
 
     > :last-child { flex-shrink: 0; }
   }
+
+  * + .form-version-def-dropdown { margin-left: 5px; }
```

Comment:
I also could have done:

```css
.form-version-def-dropdown:not(:first-child) { margin-left: 5px; }
```

I have a slight preference for my version because it avoids a nested selector.